source/partials/_additional_options_items.html.md

```diff
@@ -0,0 +1,252 @@
+
+# AdditionalOptionItems
+
+## Show
+
+### HTTP Request
+
+`GET http://staging.gojourny.com/api/v3/itinerary/<ITINERARY_ID>/additional_option_items/<ADDITIONAL_OPTION_ITEM>`
+
+### Query Parameters
+
+Parameter | Description
+--------- | -------
+ITINERARY_ID | The Itinerary ID
+ADDITIONAL_OPTION_ITEM | The date item ID
+
+> The above command returns JSON structured like this:
+
+```json
+{
+    "additional_option_items": {
+        "848063": {
+            "id": 848063,
+            "name": "Dining & Drinks",
+            "note": null,
+            "trip_section_id": null,
+            "row_order_position": 0,
+            "description": null,
+            "itinerary_id": "58Dy",
+            "location_items": [
+                {
+                    "resource_type": "location_item",
+                    "id": 848291
+                },
+                {
+                    "resource_type": "location_item",
+                    "id": 848269
+                },
+                {
+                    "resource_type": "location_item",
+                    "id": 848271
+                },
+                {
+                    "resource_type": "location_item",
+                    "id": 848094
+                },
+                {
+                    "resource_type": "location_item",
+                    "id": 848103
+                },
+                {
+                    "resource_type": "location_item",
+                    "id": 848081
+                },
+                {
+                    "resource_type": "location_item",
+                    "id": 848079
+                },
+                {
+                    "resource_type": "location_item",
+                    "id": 848101
+                },
+                {
+                    "resource_type": "location_item",
+                    "id": 848080
+                },
+                {
+                    "resource_type": "location_item",
+                    "id": 848100
+                }
+            ]
+        }
+    }
+}
+```
+
+## Create
+
+Allows the user to create a Additional Option Item
+
+### HTTP Request
+
+`POST http://staging.gojourny.com/api/v3/itinerary/<ITINERARY_ID>/additional_option_items`
```

Comment:
http://staging.gojourny.com/api/v3/itineraries/<ITINERARY_ID>/additional_option_items

Itineraries, not itinerary

Reply:
@mauu-alpha when I tested:

it required the field in the body to be `additional_option_items` (with the s). 
Two things:
1. Adding the `name` property to `additional_option_items` didn't seem to add the data.
2. I think it should be without the s, or just flatted.